ethereum_bridge/src/test_utils.rs

```diff
@@ -191,8 +189,9 @@ pub fn init_storage_with_validators(
     // set last height to a reasonable value;
     // it should allow vote extensions to be cast
     if let Some(b) = wl_storage.storage.last_block.as_mut() {
-        b.height = 3.into();
+        b.height = 0.into();
```

Comment:
vote extension validation fails when `last_height = 0`. I think semantically this is wrong, given the validation code we have for vote extensions